test/launch_node.sh

```diff
@@ -20,11 +20,10 @@ EOF
 }
 
 function start () {
-    ./nknd "$@" <<EOF
-testbed
-EOF
+    ./nknd "$@" -p $(cat ./wallet.pswd)
 }
 
+ulimit -n 10240
```

Comment:
Does 10240 works on linux? 4096 is the max you can do (without additional setup) on MacOS

Reply:
There is another hard-top limitation at /etc/security/limits.conf on linux. 
But we could not change OS limitation in this script as privileged user, it should be option of node admin.
Whatever, it is no any side-effect even though ulimit -n 10240 failed. Here just attempt to touch the limitation.